src/main/java/com/microsoft/aad/msal4j/ITenantProfile.java

```diff
@@ -0,0 +1,30 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.microsoft.aad.msal4j;
+
+import java.util.Map;
+
+/**
+ * Interface representing a single tenant profile. ITenantProfiles are made available through the
+ * {@link IAccount#getTenantProfiles()} method of an Account
+ *
+ */
+public interface ITenantProfile {
+
+    /**
+     * @return local OID
+     */
+    String getId();
```

Comment:
Both getId() and getClaims() are declared as public in the design doc, and the Android implementation makes them public as well, so I think it's meant to be exposed. Since this value is available from getClaims() anyway we could probably keep it private without changing the actual functionality, but leaving it exposed keeps it consisted with other SDKs/the design.

Reply:
yes, it is available from claims - why this particular claim should be duplicated?

Reply:
please be more specific - is "oid" claim value from id token

Reply:
please use consistent naming - here getter , name of field , description  and description in "return" parts are all different

Reply:
I am still not sure why we need to expose it here - why int can not be read from claims map direcly

Reply:
Per our messages on teams, I've removed the getID() and getTenantID() related stuff from TenantProfile, and now it only has getClaims().